src/Domain/ValueObjects/ItemQuality.cs

```diff
@@ -0,0 +1,7 @@
+namespace Domain.ValueObjects;
+
+public record ItemQuality(float Value, bool statTrak)
+{
+    float Value { get; set; } = Value;
```

Comment:
records auto generate the init-only properties, so you dont need to type them explicitly